support_model.py

```diff
@@ -44,6 +45,17 @@ def remove_slack_commands(query):
     return query
 
 
+def get_answer_if_known(query):
+    gc = gspread.service_account(filename='ods-qna-dd4a610895ad.json')
+    questions = gc.open("faq fixed").sheet1
+
+    list_of_dicts = questions.get_all_records()
+
+    for dct in list_of_dicts:
+        if query in dct['known_question']:
```

Comment:
We have second column `known_question_synonimous`, so we have to search there too. In addition, I suggest to extract this condition to function `is_query_in_row`. Further, we will improve this condition.

Reply:
- [ ] add method is_query_in_row

Reply:
- [ ] find known_question_synonimous

Reply:
What about tests?